src/V3Region.cpp

```diff
@@ -0,0 +1,102 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+// DESCRIPTION: Verilator: Assigning statements to regions
+//
+//*************************************************************************
+// V3Region's Transformations:
+//
+//  Adds region information to nodes
+//
+//*************************************************************************
+
+#include "config_build.h"
+#include "verilatedos.h"
+
+#include "V3Global.h"
+#include "V3Region.h"
+#include "V3Ast.h"
+
+#include <map>
+
+class RegionVisitor : public AstNVisitor {
+    bool m_inInactive;
+    bool m_inReactive;
```

Comment:
Now using C++11.  Also please comment, thanks.

```suggestion
    bool m_inInactive = false;
    bool m_inReactive = false;
```